mage_ai/frontend/components/datasets/FeatureProfiles/index.style.tsx

```diff
@@ -12,8 +13,8 @@ export const ContainerStyle = styled.div`
 `;
 
 export const ColumnProfileStyle = styled.div`
-  background: ${SILVER};
   ${props => `
+    background: ${props.theme.background.table};
```

Comment:
default all the props in this file to light or dark theme color like we do in other styled components